hmda/src/main/scala/hmda/api/http/filing/submissions/EditsHttpApi.scala

```diff
@@ -96,7 +96,7 @@ trait EditsHttpApi extends HmdaTimeDirectives {
 
   //institutions/<institutionId>/filings/<period>/submissions/<submissionId>/edits/edit
   def editDetailsPath(oAuth2Authorization: OAuth2Authorization): Route = {
-    val editNameRegex: Regex = new Regex("""[SVQ]\d\d\d""")
+    val editNameRegex: Regex = new Regex("""[SVQ]\d\d\d(?:-\d)*""")
```

Comment:
This could probably be simplified to `(-\d)?`, we only want to match this pattern 0 or 1 times.

Reply:
`(?:)` forms a non-capturing group. Which, from my testing, seems to be necessary for Akka PatternMatcher DSL. without the `(?:)` we just get `-2` for `editName` when, for example, `Q613-2` get passed. I have however changed `*` to `?` to capture 0 or 1

Reply:
Nice, that sounds perfect! 👍